.github/workflows/build.yml

```diff
@@ -166,9 +166,9 @@ jobs:
                config_arg: "CFLAGS='-O0'"},
               {name: 'linux-arm64', os: 'ubuntu-24.04-arm',
                'test-in-prefix': true},
-              {name: 'macos-x86_64', os: 'macos-13',
+              {name: 'macos-x86_64', os: 'macos-15-intel',
                'test-in-prefix': true},
-              {name: 'macos-arm64', os: 'macos-15',
+              {name: 'macos-arm64', os: 'macos-latest',
```

Comment:
What is macos-latest testing? (Why not stick to macos-15 there?)

Reply:
`macos-latest` is currently an alias to `macos-15`. For ARM I'm not seeing a reason to stick to a specific macOS release rather than always getting the latest. In other workflows we use `ubuntu-latest` and `windows-latest` rather than fixed version.